data-es6.js

```diff
@@ -1059,6 +1082,28 @@ exports.tests = [
       },
     },
     {
+      name: 'for-loop iteration scope',
```

Comment:
I copied this from the `let` section & just changed `let` to `const`. Should I apply those changes in both places?